src/commands/createNewProject/ProjectCreateStep/JavaScriptProjectCreateStep.ts

```diff
@@ -36,6 +40,12 @@ export class JavaScriptProjectCreateStep extends ScriptProjectCreateStep {
                 devDependencies: this.getPackageJsonDevDeps(context)
             });
         }
+
```

Comment:
I liked having it as a function still. I think it'd be nicer to have it as a private function on the class.

Also, bit of a nit, but seeing `PackageManger.npm` as the command is a bit weird to me. We both know that the package manager is `npm` which is also the name of the CLI command, but it doesn't feel very intuitive to me to pass it in as a command arg. But that's definitely a non-issue.